src/components/Helpers.js

```diff
@@ -51,9 +52,17 @@ export const formatBytes = (bytes, decimals = 2) => {
 }
 
 /** Returns a human readable representation of a number of matches **/
-export const formatMatchString = (hitCount, online) => {
+export const formatMatchString = (hitCount, online) => {  
   const count = hitCount === 10000 ? `${hitCount}+` : hitCount
-  const suffix = online ? (hitCount === 1 ? 'digital match' : 'digital matches' ) : (hitCount === 1 ? 'match' : 'matches')
+  const physicalMatch = t({
+    comment: 'Pluralization of match(es)',
+    message: plural(hitCount, {one: 'match', other: 'matches'})
+  })
+  const digitalMatch = t({
+    comment: 'Pluralization of digital match(es)',
+    message: plural(hitCount, {one: 'digital match', other: 'digital matches'})
+  })
+  const suffix = (online ? digitalMatch : physicalMatch)
```

Comment:
This is doing different conditional logic but when it pertains to the number of items. Doing it this way also allows for the pluralization to be extended for languages that have more than two types of plural forms.